BizTravel360/src/main/resources/application.properties

```
@@ -8,8 +8,20 @@ delegations.file.path=BizTravel360/src/main/resources/data/delegations.json
 #Buffer time in hours
 delegation.validation.time-buffer=1
 
+application.name = @project.name@
+application.version = @project.version@
+
+spring.datasource.url = jdbc:mysql://localhost:3306/biztravel360
+spring.datasource.username = biztravel
+spring.datasource.password = biztravel
+spring.datasource.driver-class-name = com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.format_sql=true
+spring.jpa.show-sql = true
+spring.jpa.hibernate.ddl-auto = validate
```

```suggestion
spring.jpa.hibernate.ddl-auto = update
```
nie będziemy w tym projekcie używać flywaya więc tutaj zrobimy update i oddamy updaty kolumn w ręce springa.